marimo/_runtime/runtime.py

```diff
@@ -1997,6 +2000,18 @@ async def instantiate(self, request: CreationRequest) -> None:
             for cid in self._uninstantiated_execution_requests:
                 CellOp.broadcast_stale(cell_id=cid, stale=True)
 
+    def load_dotenv(self) -> None:
+        try:
+            from dotenv import load_dotenv
+
+            for env in self.user_config["runtime"]["dotenv"]:
+                if Path(env).exists():
+                    load_dotenv(env)
+        except ImportError:
+            pass
+        except Exception:
+            LOGGER.error("Failed to load .env file")
```

Comment:
can we push the exception inside the for loop, so some env will get picked up. 

and could we also format: `LOGGER.error(f"Failed to load {env} file")`

Reply:
i guess it would be try/except (1 for the import and 1 for the forloop)